TombEngine/Specific/level.cpp

```diff
@@ -1744,22 +1793,3 @@ void BuildOutsideRoomsTable()
 		}
 	}
 }
-
-void LoadPortal(ROOM_INFO& room) 
```

Comment:
You have removed implementation, but forgot to remove declaration. Where door data is loaded now?

Reply:
Loading has been inlined.